cmd/snap-bootstrap/partition/sfdisk.go

```diff
@@ -353,29 +363,19 @@ func buildPartitionList(dl *DeviceLayout, pv *gadget.LaidOutVolume) (sfdiskInput
 			s.Label = ubuntuDataLabel
 		}
 
-		toBeCreated = append(toBeCreated, DeviceStructure{p, node})
+		toBeCreated = append(toBeCreated, DeviceStructure{p, node, true})
```

Comment:
Nitpick, this could use a keyword init now:
```
DeviceStructure{
    LaidOutStructure: p,
    Node: node,
    Created: true,
}
```

Reply:
Agree, it's way more readable.